### ISSUE TITLE
Etherpad admin settings are accessible to anyone

### ISSUE THREAD

**Issue by simtrami** [spantaleev/matrix-docker-ansible-deploy#1866]
This is probably an issue with Etherpad or the Dimension integration, I did not look much into it yet.
I just wanted to urgently warn about the fact the Etherpad admin panel becomes publicly accessible when enabled the way it is described [in the docs](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-etherpad.md#etherpad-admin-access-optional):

> Etherpad comes with a admin web-UI which is disabled by default. You can enable it by setting a username and password in your configuration file (inventory/host_vars/matrix.<your-domain>/vars.yml):
>  ```yaml
> matrix_etherpad_admin_username: admin
> matrix_etherpad_admin_password: some-password
> ```

---

**To Reproduce**
My `vars.yml` file looks like this:

```yaml
#...

matrix_homeserver_implementation: synapse

#...

# Install the Dimension integration manager
matrix_dimension_enabled: true
matrix_dimension_access_token: [secret]
matrix_dimension_admins:
  - "@[my user which was not connected to Element while testing]"

# Install a self-hosted Etherpad instance
matrix_etherpad_enabled: true
matrix_etherpad_admin_username: admin
matrix_etherpad_admin_password: [secret]
```

I use the default Nginx proxy with Let's Encrypt for SSL. Nothing fancy here.

---

Install [Dimension](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-dimension.md) + [Etherpad](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-etherpad.md) as the docs say.

Then open your self-hosted Etherpad admin panel : `https://dimension.<your-domain>/etherpad/admin` :
**you are able to see the configuration, edit it and save it without having to log in with the given credentials.**

**Expected behavior**
You should be prompted to log in with the credentials given in `vars.yml` when trying to access Etherpad's admin panel.

**Matrix Server:**
 - OS: Ubuntu 22.04 LTS
 - Architecture: amd64

**Client:**
 - Browser: Firefox
 - Version: 100.0.2 (Firefox)

**Additional context**
I don't expect it to be fixed by maintainers of this repo, I want to warn about this issue as any user of the Etherpad role might run into it.
Maybe temporarily disable the admin panel feature for new installs while it is fixed.

It might be a Nginx proxy misconfiguration though, or a Dimension one.
I need to tinker with it a bit more.

**Comment by spantaleev** [comment]
Accessible by anyone.. having the username and password, you mean?

**Comment by simtrami** [comment]
Hi, no.

Or maybe you could not reproduce?

> Then open your self-hosted Etherpad admin panel : https://dimension.<your-domain>/etherpad/admin :
> you are able to see the configuration, edit it and save it without having to log in with the given credentials.
> 
> **Expected behavior**
> You should be prompted to log in with the credentials given in vars.yml when trying to access Etherpad's admin panel.

**Comment by spantaleev** [comment]
I use neither Dimension, nor Etherpad, so I can't easily give this a try. It should be investigated though.

**Comment by simtrami** [comment]
I set up a new instance on another domain with a minimal configuration (synapse, element, dimension & etherpad) and I could NOT reproduce the bug: https://dimension.spatializer.fr/etherpad/admin
I am going to try to reproduce it like yesterday, on my main homeserver. I would be quite embarrassed if it prompted me for HTTP login, as I don't think I would be able to understand why 😕

**Comment by simtrami** [comment]
OK, I figured out what the problem was and I am sorry to have wasted your time.

I should have tested this on a vanilla browser: it was **my password manager** browser extension, Bitwarden, which filled in the HTTP login form before Firefox displayed it.
I had prepared a Bitwarden login profile for Etherpad admin during the deployment, so it never actually prompted me once.

Then I saw I could access the admin panel by just pasting the URL. Because there are some passwords accessible in clear text and the edit+save functions were possible, I must have panicked and could not think straight. I thought I was helping but false alarm: am just dumb 😐

I am closing with _can't repro_, have a nice day, sorry again.
